### PR TITLE
Fix typo in MS_MAP_PATTERN regexp

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -77,7 +77,7 @@ ENV APACHE_CONFDIR=/etc/apache2 \
     LANG=C \
     TERM=linux \
     MS_MAPFILE=/etc/mapserver/mapserver.map \
-    MS_MAP_PATTERN=^\/etc\/mapserver\/[^\.][_A-Za-z0-9\-\.]+\/{1})*([_A-Za-z0-9\-\.]+\.map$
+    MS_MAP_PATTERN=^\/etc\/mapserver\/([^\.][_A-Za-z0-9\-\.]+\/{1})*([_A-Za-z0-9\-\.]+\.(map))$
```
Comment (sbrunner) [inline Dockerfile:80]:
```suggestion
    MS_MAP_PATTERN=^\/etc\/mapserver\/([^\.][_A-Za-z0-9\-\.]+\/{1})*([_A-Za-z0-9\-\.]+\.map)$
```
The () around maps isn't needed...

**Review by sbrunner — CHANGES_REQUESTED** [review]
see comment

**Comment by yjacolin** [comment]
> see comment

done :)

**Review by sbrunner — APPROVED** [review]
Thanks :-)

**Pull request merged**